From 95e52727469d4c03464381d7b0e1649af590ba13 Mon Sep 17 00:00:00 2001
From: Ben Mares <services-git-throwaway1@tensorial.com>
Date: Fri, 25 Nov 2022 18:43:59 +0100
Subject: [PATCH] Fix AttributeError: module 'collections' has no attribute
 'abc'

See https://github.com/conda-forge/dicttoxml-feedstock/pull/5#issuecomment-1327181435
---
 dicttoxml.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/dicttoxml.py b/dicttoxml.py
index 0b517c0..a73b073 100755
--- a/dicttoxml.py
+++ b/dicttoxml.py
@@ -15,10 +15,11 @@
 version = __version__
 
 from random import randint
-import collections
 try:
+    import collections.abc
     iterable = collections.abc.Iterable
-except AttributeError:
+except ImportError:
+    import collections
     iterable = collections.Iterable
 
 import numbers
